static/app/components/events/autofix/autofixSolution.tsx

```diff
@@ -375,17 +386,37 @@ function AutofixSolutionDisplay({
     handleAddInstruction();
   };
 
-  const handleDeleteItem = useCallback((index: number) => {
-    setSolutionItems(current => current.filter((_, i) => i !== index));
-  }, []);
+  const handleDeleteItem = useCallback(
+    (index: number) => {
+      setSolutionItems(current => current.filter((_, i) => i !== index));
 
-  const handleToggleActive = useCallback((index: number) => {
-    setSolutionItems(current =>
-      current.map((item, i) =>
-        i === index ? {...item, is_active: item.is_active === false ? true : false} : item
-      )
-    );
-  }, []);
+      trackAnalytics('autofix.solution.delete_step', {
+        organization,
+        solution: solutionItems,
+        deletedStep: solutionItems[index],
+      });
+    },
+    [organization, solutionItems]
+  );
+
+  const handleToggleActive = useCallback(
+    (index: number) => {
+      setSolutionItems(current =>
+        current.map((item, i) =>
+          i === index
+            ? {...item, is_active: item.is_active === false ? true : false}
+            : item
+        )
+      );
+
+      trackAnalytics('autofix.solution.toggle_step', {
```

Comment:
Name should probably be more clear that this is disabling the agent's step vs deleting the human's own step